assets/js/googlesitekit/datastore/user/authentication.test.js

```diff
@@ -430,5 +431,50 @@ describe( 'core/user authentication', () => {
 				expect( unsatisfiedScopes ).toEqual( undefined );
 			} );
 		} );
+
+		describe( 'needsReauthentication', () => {
```

Comment:
This selector should also have the "uses a resolver get all authentication info" test like the other ones do.

Reply:
Test added